### DISCUSSION TITLE
Pop

### DISCUSSION THREAD

**Discussion by SussyXD** [iptv-org/community#677] [Q&A]
**Channel Name: POP
**Channel ID: Pop.uk
**Country: UK
**Language: English
**External Link (optional):
**Notes (optional): Free to air UK channel

**Comment by UltraHDR** [comment]
http://hybridtv.ss7.tv/whitelabel/index.html?env=prd&platform=fvp&cid=sonykids&platformenv=prd&service=freeviewplay

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/uk.m3u#L131-L134
